PlayCover/ViewModel/UninstallVM.swift

```diff
@@ -0,0 +1,22 @@
+//
```

Comment:
I don't think the uninstaller needs the non-blocking function, as the cache clearing function is now pretty fast due to a bug fix (<2 seconds for Genshin).